### PR TITLE
Resolve comments mike-ivs July part I

### LINKED ISSUE #483: Improve questions and objectives episode 1 & 2
I think the Episode 1 & 2 "questions & objectives" may have gotten a bit mixed up at some point.
Questions: "what is a neural network" should be in this episode instead of episode 2
Objectives: "Identify the inputs and outputs of a deep neural network." should be moved to episode 2
Questions and objectives could be re-ordered chronologically according to when they are covered in the episode (the other episodes do this I believe)

### LINKED ISSUE #484: Fix figure `fig/01_AI_ML_DL_differences.png`
- The alt text of fig/01_AI_ML_DL_differences.png uses the acronyms AI/ML/DL/NN. NN is the only one that isn't defined in the episode, so worth defining somewhere, or just expanding out the acronyms.
- The cited source for fig/01_AI_ML_DL_differences.png doesn't seem to line up with that specific image. I think it might belong to Intel according to [this source that cites it](https://www.anandtech.com/show/12673/titan-v-deep-learning-deep-dive/2), the [Intel Introduction to AI course Week 1](https://www.intel.com/content/www/us/en/developer/topic-technology/artificial-intelligence/training/course-artificial-intelligence.html) slides, and the colour palette compared to Intels other infographic style icons. But i'm not certain...
- Is it worth putting in a generative AI subset within the DL circle, like [this example](https://geetikakaushik2020.medium.com/unveiling-the-landscape-of-generative-ai-and-language-models-de0167d21f64)?

### LINKED ISSUE #491: Move episode 3 dataset download in the instructor notes to the episode itself and make it viewable by learners

### PR DESCRIPTION
**Pull request by svenvanderburg**
Fixes some tasks from #482 

Fixes #491
Fixes #483 
Fixes #484

### CODE REVIEW

File: episodes/3-monitor-the-model.Rmd [hunk 1]
```diff
@@ -55,6 +55,14 @@ years 2000 to 2010. For all locations the data contains the variables ‘mean te
 ### Import Dataset
 We will now import and explore the weather data-set:
 
+::: callout
```
Comment (svenvanderburg) [inline episodes/3-monitor-the-model.Rmd:58]: I moved this here so it is clearer

File: instructors/instructor-notes.md [hunk 2]
```diff
@@ -14,11 +14,6 @@ The episodes are quite long, because they cover a full cycle of the deep learnin
 ## Episode 3: Monitor the training process
 When episode 3 is taught on a different day then episode 2, it is very useful to start with a recap of episode 2. The Key Points of episode 2 can be iterated, and you can go through the code of the previous session (without actually running it). This will help learners in the big exercise on creating a neural network.
 
-If learners did not download the data yet, they can also load the data directly from zenodo (instead of first downloading and saving):
```
Comment (svenvanderburg) [inline instructors/instructor-notes.md:17]: This is already clear from the episode, so removing the redundancy.

File: episodes/1-introduction.Rmd [hunk 3]
```diff
@@ -7,24 +7,26 @@ exercises: 30
...
-- How do neural networks learn?
 
 ::::::::::::::::::::::::::::::::::::::::::::::::
 
 :::::::::::::::::::::::::::::::::::::: objectives
 
+- Define deep learning
+- Describe how a neural network is build up
+- Explain the operations performed by a single neuron
+- Describe what a loss function is
```
Comment (carschno) [inline episodes/1-introduction.Rmd:25]: Minor remark, as we are re-thinking objectives: are these really objectives? I would see it rather along these lines:
```suggestion
- Lear the definition of deep learning
- Understand how a neural network is build up
- Learn about the operations performed by a single neuron
- Understand what a loss function is
```
Anyway, I think it is fine as it is. I just wanted to share my thoughts on the question of what an objective really is.

File: episodes/fig/01_AI_ML_DL_differences.jpg
Comment (carschno) [inline episodes/fig/01_AI_ML_DL_differences.jpg:1]: Shouldn't there be a reference to this file somewhere, that needs to be adapted? Assuming that the png or svg version was used before.

**Review by carschno — COMMENTED** [review]
Looks good!
I have had two comments, you need to check if they really are applicable.

File: episodes/1-introduction.Rmd (hunk 3 above)
Comment (svenvanderburg) [reply episodes/1-introduction.Rmd:25]: This is on purpose 😂 Carpentries-style. Starts with verbs from the Bloom taxonomy. But I agree it seems weird if you don't know about it.

File: episodes/fig/01_AI_ML_DL_differences.jpg
Comment (svenvanderburg) [reply episodes/fig/01_AI_ML_DL_differences.jpg:1]: Yes, it was changed in episode 1. Which is the only reference.

**Pull request merged**